gui/src/main.rs

```diff
@@ -119,8 +119,8 @@ impl Application for GUI {
 
     fn title(&self) -> String {
         match self.state {
-            State::Installer(_) => String::from("Liana Installer"),
-            _ => String::from("Liana"),
+            State::Installer(_) => format!("Liana v{}.{} Installer", VERSION.major, VERSION.minor),
+            _ => format!("Liana v{}.{}", VERSION.major, VERSION.minor),
```

Comment:
`Version` already implements `Display`. Let's use the existing implementation instead of reimplementing it:
```suggestion
            State::Installer(_) => format!("Liana v{} Installer", VERSION),
            _ => format!("Liana v{}", VERSION),
```

This also carries the update used for dev branches vs release branches.

Reply:
done